### PR TITLE
Add styles:

### PR DESCRIPTION
**Pull request by fanxhe-pair**
-Margins
-Style button
-Change dimensions of the text-area and iframe--container.

### CODE REVIEW

File: public/assets/css/module.css [hunk 1]
```diff
@@ -1,16 +1,15 @@
-
 .code-area {
   background-color: #333333;
   color: #fff;
+  height: 77% !important;
```
Comment (ingilniero) [inline public/assets/css/module.css]: Is this `!important` really needed?, I mean if we can get rid of it. If this class is overriding styles from a bootstrap class for example, maybe we should check if we really need that bootstrap class, if so, what do you think about making a selector with higher specificity to override the bootstrap styles?.

**Comment by ingilniero** [comment]
Besides my comment, LGTM. Really great job :+1:

File: public/assets/css/module.css (hunk 1 above)
Comment (fanxhe-pair) [inline public/assets/css/module.css]: We do it to override bootstrap class property. How can we override it with another selector?

Comment (ingilniero) [inline public/assets/css/module.css]: Adding a selector with a higher specificity will be enough, supposing we want override the bootstrap's class `form-control` , we'd need something like:

```
form-control.code-area {
 ...

}
```

This CSS selector will be targeting an element that will need to have both classes, being more specific than an element that just has one of the two classes.

Comment (anxhe) [inline public/assets/css/module.css]: :+1:

**Comment by joiggama** [comment]
:+1:

**Pull request merged**